File: src/kyu_8/plural/java/PluralTest.java
package kyu_8.plural.java;

import org.junit.Test;

import static org.junit.Assert.assertEquals;

public class PluralTest {
    @Test
    public void BasicTest() {
        assertEquals(true, Plural.isPlural(0f));
        assertEquals(true, Plural.isPlural(0.5f));
        assertEquals(false, Plural.isPlural(1f));
        assertEquals(true, Plural.isPlural(100f));
    }
}

File: src/kyu_8/plural/js/pluralTest.js
const Test = require('@codewars/test-compat');
const plural = require("./plural");

describe("Tests", () => {
    it("test", () => {
        Test.assertEquals(plural(0), true, "Plural for 0");
        Test.assertEquals(plural(0.5), true, "Plural for 0.5");
        Test.assertEquals(plural(1), false, "Plural for 1");
        Test.assertEquals(plural(100), true, "Plural for 100");
        Test.assertEquals(plural(Infinity), true, "Plural for Infinity");
    });
});
